docker/aarch64/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:22.04 as builder
+
+ENV LANG en_US.utf8
+WORKDIR /greptimedb
+
+# Install dependencies.
+RUN apt-get update && apt-get install -y \
+    libssl-dev \
+    protobuf-compiler \
+    curl \
+    build-essential \
+    pkg-config \
+    wget
+
+# Install Rust.
+SHELL ["/bin/bash", "-c"]
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --no-modify-path --default-toolchain none -y
+ENV PATH /root/.cargo/bin/:$PATH
+
+# Install cross platform toolchain
+RUN apt-get -y update && \
+    apt-get -y install libssl-dev pkg-config g++-aarch64-linux-gnu gcc-aarch64-linux-gnu && \
+    apt-get install binutils-aarch64-linux-gnu
+
+COPY . .
+# This three env var is set in script, so I set it manually in dockerfile.
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/
+ENV LIBRARY_PATH=$LIBRARY_PATH:/usr/local/lib/
+ENV PY_INSTALL_PATH=${PWD}/python_arm64_build
+RUN chmod +x ./docker/aarch64/compile-python.sh && \
+    ./docker/aarch64/compile-python.sh
+# Install rustup target for cross compiling.
+RUN rustup target add aarch64-unknown-linux-gnu
+# Set the environment variable for cross compiling and compile it
+# Build the project in release mode. Set Net fetch with git cli to true to avoid git error.
+RUN export PYO3_CROSS_LIB_DIR=$PY_INSTALL_PATH/lib && \ 
+    alias python=python3 && \
+    CARGO_NET_GIT_FETCH_WITH_CLI=1 && \
+    cargo build --target aarch64-unknown-linux-gnu --release
```

Comment:
Do we need to enable `pyo3`'s feature gate in this build command?

Reply:
Yes

> Do we need to enable `pyo3`'s feature gate in this build command?

Thanks for noticing, running testing on it.